proto/tendermint/types/types.proto

```diff
@@ -215,9 +216,9 @@ message BlockMeta {
 
 // TxProof represents a Merkle proof of the presence of a transaction in the Merkle tree.
 message TxProof {
-  bytes                   root_hash = 1;
-  bytes                   data      = 2;
-  tendermint.crypto.Proof proof     = 3;
+  repeated bytes    row_roots = 1;
+  repeated bytes    data      = 2;
+  repeated NMTProof proofs    = 3;
 }
```

Comment:
It's possible for a transaction to span multiple rows, and therefore we now need to potentially prove two or more rows